table/sst_file_reader_test.cc

```diff
@@ -307,6 +308,25 @@ class SstFileReaderTimestampTest : public testing::Test {
     ASSERT_OK(iter->status());
   }
 
+  void CheckFile(const std::string& timestamp, const OutputKeyValueDesc& desc) {
+    SstFileReader reader(options_);
+
+    ASSERT_OK(reader.Open(sst_name_));
+    ASSERT_OK(reader.VerifyChecksum());
+
+    Slice ts_slice(timestamp);
+
+    ReadOptions read_options;
+    read_options.timestamp = &ts_slice;
+
+    std::unique_ptr<Iterator> iter(reader.NewIterator(read_options));
+    iter->Seek(desc.key);
+    ASSERT_TRUE(iter->Valid());
+    ASSERT_EQ(iter->key(), desc.key);
+    ASSERT_EQ(iter->timestamp(), desc.timestamp);
+    ASSERT_EQ(iter->value(), desc.value);
+  }
+
```

Comment:
Might have missed something obvious but is this new `CheckFile` variant used somewhere?

Reply:
Thank you for catching this! This is indeed obsolete.